### ISSUE TITLE
Convert Tutorials HTML page to a PHP page that pulls data from backend database

### ISSUE THREAD

**Issue by mutharasus** [AccessibleAI-edu/accessibleai-edu#102]

**Comment by MarijaIgnj** [comment]
@mutharasus This is all good, we are just missing the download PDF button in both Tutorials. Also, can we either put Accessible AI as an author or just erase the author option generally.

**Comment by mutharasus** [comment]
Yes Marija, this being the test environment it shows as Admin. In production we will stick to 2 users. One for yourself and one for Arjun. Whoever creates the post / tutorial / Multimedia will have their name show up in the production site. We can add a future feature where if you click on the name then it shows all the content created by that user.

Download PDF button is now fixed in Tutorials and Multimedia

**Comment by MarijaIgnj** [comment]
@mutharasus The thing is, as I wrote here, that tutorials and multimedia do not need our names, but something general - just Accessible AI
